addon/mixins/axis-titles.js

```diff
@@ -0,0 +1,174 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
```

Comment:
I think we should define a local variable with an understandable name before exporting it. 
```
const AxisTitlesMixin = Ember.Mixin.create({
  ...
});
export default AxisTitlesMixin;
```